components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -321,6 +332,8 @@ message ControlItem {
 	google.protobuf.Timestamp end_time = 5;
 	// Intentionally blank.
 	ControlSummary control_summary = 6;
+	// An array returns how many nodes have been waived or not
```

Comment:
```suggestion
	// A list of waivers for the nodes affected by this control.
```

I'm not sure if this suggestion makes any sense but the original comment was not clear either.